spec/datadog/tracing/contrib/concurrent_ruby/integration_test_spec.rb

```diff
@@ -135,6 +135,18 @@
           end
         end
       end
+
+      context 'when propagates without an active trace' do
+        it 'creates a root span' do
```

Comment:
Fix looks good but that sounds weird. I would expect:

- 'does nothing'
- 'does not raise'
- 'calls executor'